bemani/utils/read.py

```diff
@@ -3473,6 +5313,67 @@ def import_music_db(self, songs: List[Dict[str, Any]]) -> None:
             jubeat.import_emblems(emblems)
         jubeat.close()
 
+    elif args.series == GameConstants.GITADORA:
+        gtd = ImportGitadora(config, args.version, args.no_combine, args.update)
+        if args.tsv is not None:
+            # Special case for Jubeat, grab the title/artist metadata that was
+            # hand-populated since its not in the music DB.
+            #gtd.import_metadata(args.tsv)
+            print("GITADORA does not require that you import a TSV.")
+        else:
+            # Normal case, doing a music DB or emblem import.
+            if args.xml is not None:
+                songs = gtd.scrape(args.xml)
+            elif args.server and args.token:
+                songs = gtd.lookup(args.server, args.token)
+            else:
+                raise Exception(
+                    'No music_info.xml or TSV provided and no remote server specified! Please ' +
+                    'provide either a --xml, --tsv or a --server and --token option!'
+                )
+            gtd.import_music_db(songs)
+        gtd.close()
+    
+    elif args.series == GameConstants.DRUMMANIA:
+        dm = ImportDrummania(config, args.version, args.no_combine, args.update)
+        if args.tsv is not None:
+            # Special case for Jubeat, grab the title/artist metadata that was
+            # hand-populated since its not in the music DB.
+            dm.import_metadata(args.tsv)
+        else:
+            # Normal case, doing a music DB or emblem import.
+            if args.xml is not None:
+                songs = dm.scrape(args.xml)
+            elif args.server and args.token:
+                songs = dm.lookup(args.server, args.token)
+            else:
+                raise Exception(
+                    'No music_info.xml or TSV provided and no remote server specified! Please ' +
+                    'provide either a --xml, --tsv or a --server and --token option!'
+                )
+            dm.import_music_db(songs)
+        dm.close()
+
+    elif args.series == GameConstants.GUITARFREAKS:
+        gf = ImportGuitarFreaks(config, args.version, args.no_combine, args.update)
+        if args.tsv is not None:
+            # Special case for Jubeat, grab the title/artist metadata that was
+            # hand-populated since its not in the music DB.
+            gf.import_metadata(args.tsv)
+        else:
+            # Normal case, doing a music DB or emblem import.
```

Comment:
This looks completely copypasta, and there's no tsv for gfdm songs, why is a lot of this even here?

Reply:
agan, GFDM requires a TSV, like jubeat/iidx